update_mkdocs_yml.py

```diff
@@ -15,7 +15,7 @@
     mkdocs = yaml.load(f, Loader=yaml.SafeLoader)
 
 mkdocs["site_url"] = site_url
-mkdocs["edit_uri"] = 'edit/master/{}/'.format(docs_dir)
+mkdocs["edit_uri"] = 'edit/HEAD/{}/'.format(docs_dir)
```

Comment:
Unfortunately this does not work for editing: https://github.com/laminas/laminas-inputfilter/edit/HEAD/docs/book/intro.md

It must be: https://github.com/laminas/laminas-inputfilter/edit/2.19.x/docs/book/intro.md

Reply:
@froschdesign  Ok, I undid that commit for now and restored it to fixing the URLs in the footer only.

PR [#61](https://github.com/laminas/documentation-theme/pull/61) seems to be created to deal with the `edit_uri` and specific version branch if I'm right?